test/run-tests.sh

```diff
@@ -12,4 +12,4 @@
 
 cd "$(dirname "$0")"
 
-py.test -v --junit-xml=test_result.xml
+py.test-3 -v --junit-xml=test_result.xml
```

Comment:
Will this work on all distributions?
It might be, but my gut feeling tells me that it is different...

Either way I like the idea of enforcing the use of Python3

Reply:
well, google confirms that the binary seems to be called `py.test-3` everywhere.